base/doc/fntguide.tex

```diff
@@ -2170,8 +2170,259 @@ \section{Additional text symbols -- \textsf{textcomp}}
     \verb|\texttimes|                & \texttimes                & 214 \\
     \verb|\textdiv|                  & \textdiv                  & 246
   \end{tabular}
-\caption{Text symbols formerly from the \textsf{textcomp} package}
-\label{tab:textcomp}
+  \caption{Text symbols formerly from the \textsf{textcomp} package}
+  \label{tab:textcomp}
+\end{table}
+
+The |TS1| encoding contains a rich set of symbols which means that
+several symbols are only available in a few \TeX{} fonts and some, such
+as the capital accents, not available at all but developed as part of
+the reference font implementation.  In reality, many existing fonts
+don't provide a full set of glyphs defined in |TS1| encoding and the
+question arises: ``Which glyphs of the |TS1| encoding are implemented by
+which font?''
+
+\NEWfeature{2021/06/01}
+Fonts can be ordered in sub-encodings with the |\DeclareEncodingSubset|
+macro:
+\begin{decl}
+  |\DeclareEncodingSubset| \arg{encoding}
+                           \arg{font family}
+                           \arg{subset number}
+\end{decl}
+The macro takes 3 mandatory arguments: An \m{encoding} for which a
+subsetting is wanted (currently only |TS1|), the \m{font family} for
+which we declare the subset and finally the \m{subset number} between
+|0| (all of the encoding is supported) and |9| (many glyphs are
+missing).  Hence, it is assumed that some symbols are always available
+by all fonts and each sub-encoding defines macros which become
+unavailable (i.e., they are not provided in the subencoding with that
+number and all sub-encodings with higher numbers.)
+
+Thus the symbols that are available in sub-encoding $x$ are the symbols
+in table~\ref{tab:ts1-subset-always} (always available) and the symbols
+list in sub-encodings $x + 1$ and higher.  The tables
```

Comment:
that's the wrong way around \textflorin has encoding 6 which means it is available in 0-5

Reply:
This text is actually a verbatim copy from [lttextcomp.dtx](https://github.com/latex3/latex2e/blob/c66a033aaa6efdc05203580ee7e395061d6b9ab1/base/lttextcomp.dtx#L478).  Did I get it wrong?

Reply:
@dr-scsi maybe more that I got it wrong in that file. The paragraph above the one you refer to says the opposite and that one is correct. I think I need to look at that text there more closely.

Reply:
@dr-scsi take a look at what it nows says in lttextcomp if that is more understandable